test/bootstrap.sh

```diff
@@ -1,26 +1,40 @@
 #!/bin/bash
+# use bash ./bootstrap.sh, NOT sh ./bootstrap.sh,
 
 CYAN='\033[1;36m'
 NC='\033[0m'
 
 trap ctrl_c INT
-function ctrl_c {
+ctrl_c () {
     exit 11;
 }
 
 RECOMPILE_AND_RESET_EOSIO_CONTRACTS=0
-EOSIO_CONTRACTS_ROOT=/home/kedar/eosio.contracts/build/contracts/
+REBUILD_EVERIPEDIA_CONTRACTS=1
+RESET_NODEOS=1
+#EOSIO_CONTRACTS_ROOT=/home/kedar/eosio.contracts/build/contracts/
+EOSIO_CONTRACTS_ROOT=/home/travis/Programs/contracts/eosio.contracts/build/contracts
```

Comment:
Should you use `$USER` instead of hardcoding `travis`?

Reply:
will do